backend/src/extension.ts

```diff
@@ -129,7 +130,7 @@ export class YeomanUIPanel {
 			vscodeYouiEvents, 
 			outputChannel, 
 			this.logger, 
-			{genFilter: this.genFilter, messages: this.messages, defaultNpmPaths},
+			{genFilter: this.genFilter, messages: this.messages, data: _.get(uiOptions, "data", {}), defaultNpmPaths},
```

Comment:
can change:
this.genFilter = GeneratorFilter.create(_.get(uiOptions, "filter"));
to:
this.genFilter = GeneratorFilter.create(_.get(uiOptions, "genFilter"));

Reply:
Same,
For the first time UI command passes only filter not genFilter